Ginger/Ginger/App.xaml.cs

```diff
@@ -927,12 +927,14 @@ private static void LoadRecentBusinessFlow()
                 }
 
                 //load new Business Flow as default
+               // TODO: Need to check is actually below LoadDefaultBussinessFlow()method is required here
                 App.BusinessFlow = LoadDefaultBusinessFlow();
```

Comment:
LoadRecentBusinessFlow() indeed needs to be changed now that we don't have Automate tab.
please change it as following: 
check if there are more than 0 BF's if yes so set App.BusinessFlow with first BF in list, if not having so use LoadDefaultBusinessFlow() to add defualt BF and set it to  App.BusinessFlow

Reply:
it is already handled in code. 
![image](https://user-images.githubusercontent.com/41137863/46740220-ed7cd080-ccbf-11e8-9fcf-d44b8e3bc06e.png)

Removing the TODO comment.